main/src/vtr_tactic/src/tactic.cpp

```diff
@@ -165,7 +165,7 @@ bool Tactic::routeCompleted() const {
   auto lock = chain_->guard();
   const auto translation = chain_->T_leaf_trunk().r_ab_inb().norm();
 
-  if (chain_->trunkSequenceId() < (chain_->sequence().size() - 1)) {
+  if (chain_->trunkSequenceId() < (chain_->sequence().size() - 2)) {
```

Comment:
Why the change?

Reply:
I was finding that this wouldnt trigger the end of path until the robot had localized to the last frame which only occured if the robot had actually moved past the last frame. This meant it basically always ignored the route_completion_threshold check that follows and wasnt reliably ending the repeats when the controller got to the end. I found switching to second to last worked much better.